lib/rcrm/types/source/sources_filter.rb

```diff
@@ -0,0 +1,9 @@
+require 'time'
```

Comment:
You're not doing any comparisons with time so you don't need to `require 'time'`

Reply:
Changed! The problems of copy and paste :)